lib/ckan_client.py

```diff
@@ -0,0 +1,63 @@
+from .util import ckan_auth_api
```

Comment:
I think i'd have the file just called `client.py` after all this is already inside a package called ckan-sdk or similar